static_analyzer/analysis_cache.py

```
@@ -436,9 +457,190 @@ def merge_results(cached_result: dict[str, Any], new_result: dict[str, Any]) ->
+        src_node = call_graph.nodes.get(src_name)
+        dst_node = call_graph.nodes.get(dst_name)
+        if src_node is None or dst_node is None:
+            continue
+
+        checked += 1
+        refs = references_cache.get(dst_name)
+        if refs is None:
+            try:
+                engine_client.did_open(Path(dst_node.file_path), adapter.language_id)
+                refs = engine_client.references(Path(dst_node.file_path), dst_node.line_start - 1, dst_node.col_start)
+            except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)